app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -100,7 +97,7 @@
         float: none
     > ul.drop-down--projects
       width: 400px
-      left: 0
+      left: -2px
```

Comment:
Why is this necessary? Is there a way to avoid it?

Reply:
I thought it was prettier  to have the 2px grey border of the drop down menu align be "outside" so that the white of the drop-down aligns with the hover background of the top menu item. I felt that the border has more of a shadow and thus belongs outside. Remember that before there was no hamburger icon and the issue was not so apparent as the border was directly hitting the viewport edge. Anyways, if you guys think it does not improve anything feel free to kill it.